tests/fuzz/runFuzzTest.sh

```diff
@@ -47,8 +47,8 @@ UBSAN_OPTIONS+="print_stacktrace=1"
 NUM_CPU_THREADS=$(nproc)
 LIBFUZZER_ARGS+="-timeout=5 -max_len=4096 -print_final_stats=1 -jobs=${NUM_CPU_THREADS} -workers=${NUM_CPU_THREADS} -max_total_time=${FUZZ_TIMEOUT_SEC}"
 
-TEST_SPECIFIC_OVERRIDES="${PWD}/LD_PRELOAD/${TEST_NAME}_overrides.so"
-GLOBAL_OVERRIDES="${PWD}/LD_PRELOAD/global_overrides.so"
+TEST_SPECIFIC_OVERRIDES="${PWD}/../../build/lib/lib${TEST_NAME}_overrides.so"
```

Comment:
Not a fan of the relative path approach, build might not always be two levels up- are there any (CMake) environment vars that could replace `${PWD}/../..`, similar to [CMAKE_SOURCE_DIR](https://cmake.org/cmake/help/latest/variable/CMAKE_SOURCE_DIR.html#variable:CMAKE_SOURCE_DIR)?

Reply:
I agree. I updated it so now cmake will pass the [CMAKE_SOURCE_DIR](https://cmake.org/cmake/help/latest/variable/CMAKE_SOURCE_DIR.html#variable:CMAKE_SOURCE_DIR) to `runFuzzTest.sh` and it will reference that path to get to the build directory.